sherpa/optmethods/src/_saoopt.cc

```diff
@@ -833,10 +835,11 @@ static PyObject* py_minim( PyObject* self, PyObject* args,
   PyObject* py_function=NULL;
   DoubleArray par, step, lb, ub;
   IntArray finalsimplex;
-  int verbose, maxnfev, nfev, iquad, ierr, initsimplex;
+  int reflect, verbose, maxnfev, nfev, iquad, ierr, initsimplex;
   double fval, ftol, simp;
 
-  if ( !PyArg_ParseTuple( args, (char*) "iiiiddO&O&O&O&O",
+  if ( !PyArg_ParseTuple( args, (char*) "iiiiiddO&O&O&O&O",
```

Comment:
```suggestion
  if ( !PyArg_ParseTuple( args, (char*) "piiiiddO&O&O&O&O",
```

The p argument means the python value is treated as a boolean - see https://docs.python.org/3/c-api/arg.html#other-objects - and the result for `reflect` will be 0 or 1. It is possible that the second `i` could also be made a `p` as I assume `verbose` is also a boolean flag.